jm-leg-cit-3-rechtsquellenverzeichnis-literaturverzeichnis/style_test028.txt

```diff
@@ -0,0 +1,120 @@
+>>===== MODE =====>>
+all
+<<===== MODE =====<<
+
+>>===== KEYS =====>>
+[
+  "U4PGJ8LG"
+]
+<<===== KEYS =====<<
+
+>>===== DESCRIPTION =====>>
+AT Case with parallel citations
+<<===== DESCRIPTION =====<<
+
+>>===== RESULT =====>>
+FIRST
+  OGH 2 Ob 328/97t.
+FIRST w/LOCATOR
+  <i>Wilhelm</i>, ecolex 1998, 692 (123)
+FIRST w/LABEL
+  <i>Wilhelm</i>, ecolex 1998, 692 Abs 123
+BIBLIOGRAPHY
+<div class="csl-bib-body">
+  <div class="csl-entry">OGH 12.02.1998, 2 Ob 328/97t = SZ 71/21 = JBl 1999, 54 (<i>Karollus</i>) = ecolex 1998, 692 (<i>Wilhelm</i>).</div>
```

Comment:
This doesn’t work as expected. I only get the last of the parallel cites:
```<div class="csl-entry">ogh 12.02.1998, 2 Ob 328/97t = ecolex 1998, 692 (<i>Wilhelm</i>).</div>```

Even worse, in Word I get this in the bibliography (instead of the bibliography line as in the test fixture):
![image](https://user-images.githubusercontent.com/214002/64908071-923b3580-d6fb-11e9-9c6c-2f652fab5fe0.png)
and this in the footnote:
![image](https://user-images.githubusercontent.com/214002/64908093-bac32f80-d6fb-11e9-8bdd-d788c9d33b82.png)
I’d expect only `OGH 2 Ob 328/97t.` in the footnote. The reporters should only be added if there were any locators.

Reply:
So, I found out that in Word I used the wrong citations. What I get now is almost as expected in the bibliography:
![image](https://user-images.githubusercontent.com/214002/64946053-4b238080-d872-11e9-8c99-89fa396425a1.png)
Only the delimiters aren’t quite right. There is `., = ` between the parallel citations. I have no idea, where the comma comes from, the period is most likely the layout suffix.

For the citation, there’s probably a conceptual issue. I need only the common part for all parallel citations if there’s no locator, so only the first one prints anything. But I need all of them in the bibliography.